genesyscloud/tfexporter/resource_genesyscloud_tf_export_test.go

```diff
@@ -1,4 +1,4 @@
-package genesyscloud
+package tfexporter
```

Comment:
In `.github/workflows/test.yml` the command is excluding these tests since they are in a sub directory. Also, I tried running them locally on this branch, and the tests claimed to be passing on the final line but I could see they weren't (`TestAccResourceTfExportUserPromptExportAudioFile` for example had path issues to the test wav files.)